From c578262d8816d27ab5530696d7b5f1e102e3b977 Mon Sep 17 00:00:00 2001
From: Denis V. Lunev <den@openvz.org>
Date: Wed, 1 Oct 2008 12:06:39 +0400
Subject: [PATCH] net: NETIF_F_VIRTUAL intersects with NETIF_F_LRO

Fortunately, this is not a part of user/kernel interface
[xemul picked 2.6.27's 4826fea3]

Signed-off-by: Denis V. Lunev <den@openvz.org>
Signed-off-by: Pavel Emelyanov <xemul@openvz.org>
---
 include/linux/netdevice.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/linux/netdevice.h b/include/linux/netdevice.h
index 2e2acdb..0a4fb43 100644
--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -525,8 +525,8 @@ struct net_device
 #define NETIF_F_LRO		32768	/* large receive offload */
 
 	/* Segmentation offload features */
-#define NETIF_F_GSO_SHIFT	16
-#define NETIF_F_GSO_MASK	0xffff0000
+#define NETIF_F_GSO_SHIFT	20
+#define NETIF_F_GSO_MASK	0xfff00000
 #define NETIF_F_TSO		(SKB_GSO_TCPV4 << NETIF_F_GSO_SHIFT)
 #define NETIF_F_UFO		(SKB_GSO_UDP << NETIF_F_GSO_SHIFT)
 #define NETIF_F_GSO_ROBUST	(SKB_GSO_DODGY << NETIF_F_GSO_SHIFT)
-- 
1.6.0.6

